typos.toml

```diff
@@ -0,0 +1,80 @@
+[files]
+extend-exclude = ["crates/resolver-tests/*", "LICENSE-THIRD-PARTY"]
+
+[default]
+extend-ignore-re = [
+  # Handles git short SHA-1 hashes
+  "[a-f0-9]{8,9}",
+
+  # Handles ssh keys
+  "AAAA[0-9A-Za-z+/]+[=]{0,3}",
+
+  # Handles paseto from login tests
+  "k3[.](secret|public)[.][a-zA-Z0-9_-]+",
+]
+
+[default.extend-identifiers]
+# This comes from `windows_sys`
+ERROR_FILENAME_EXCED_RANGE = "ERROR_FILENAME_EXCED_RANGE"
+
+# Name of a dependency
+flate2 = "flate2"
+
+# `gix` uses `rela_*`
+rela_path = "rela_path"
+rela_pkg_root = "rela_pkg_root"
+rela_root = "rela_root"
```

Comment:
Should we instead bless `rela`?

Reply:
The regex ignoring small words takes care of this now